### ISSUE TITLE
Plans to support a CI command/output somehow?

### ISSUE THREAD

**Issue by queso** [xdissent/meteor-cucumber#4]
I love the progress of this, and perhaps CI output is more a concern for the velocity team (https://github.com/xolvio/velocity/issues/14), but have you thought through this at all?

**Comment by xdissent** [comment]
Yeah! I've got some ideas and ran a few experiments this week. I plan on making some suggestions to the velocity team tomorrow on the conference call regarding CI and better velocity integration in general.

**Comment by queso** [comment]
YAY!

**Comment by SamHatoum** [comment]
PS You're on the velocity team :)
